src/components/shared/page/Banner.tsx

```diff
@@ -0,0 +1,40 @@
+import { usePathname, useRouter } from 'next/navigation';
+import React from 'react';
+import styled from 'styled-components';
+
+import NoticeRemovable from '@components/ui/NoticeRemovable';
+
+const NoticeRemovableSticky = styled(NoticeRemovable)`
+  position: sticky;
+  top: 0;
+  width: 100%;
+  z-index: 1000;
+`;
+
+const Banner: React.FC = () => {
+  const router = useRouter();
+  const currentUrl = usePathname();
+
+  if (
+    process.env.NEXT_PUBLIC_FLAG_ENABLE_COMPARATEUR === 'true' &&
+    currentUrl &&
+    !currentUrl?.startsWith('/outils/comparateur-performances')
```

Comment:
```suggestion
    !currentUrl.startsWith('/outils/comparateur-performances')
```

A priori pas besoin du ?

Reply:
Si car currentUrl est `undefined` server side (merci Next)